translations/trans_7_10_17.csv

```diff
@@ -0,0 +1,613 @@
+Key,EN,ES
```

Comment:
I would not commit this file, it's temporary and going to get out of date quickly